### ISSUE TITLE
empty string attributes

### ISSUE THREAD

**Issue by goto-bus-stop** [shuhei/pelo#14]
When doing something like:

```js
var html = typeof window !== 'undefined' ? require('bel') : require('pelo')
var value = ''
var input = html`
  <input value=${value} autofocus />
`
```

where `value` may be empty, pelo concatenates it to `<input value= autofocus />`. browsers interpret that as `<input value="autofocus" />`.

Ideally pelo would somehow detect this case and add `""` so that the output is `<input value="" autofocus />`.

**Comment by goto-bus-stop** [comment]
Oh; a related issue:

```js
html`<input value=${'x y z'} />`
```

Results in `value="x y z"` with bel, but `value=x y z` with pelo. I think a decent way to mitigate this would be to check if the character just before the interpolated value is `=`, and if so, add quotes around the value.

**Comment by shuhei** [comment]
Published as 0.1.0.
